deployment/ib-kubernetes-ds.yaml

```diff
@@ -0,0 +1,115 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ib-kubernetes
+  namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: ib-kubernetes
+rules:
+  - apiGroups: [""]
+    resources: ["pods"]
+    verbs: ["get", "list", "patch", "watch"]
+  - apiGroups: ["k8s.cni.cncf.io"]
+    resources: ["*"]
```

Comment:
here we just need `NetworkAttachmentDefinition` right ?

can be kept as is, not critical imo...